internal/homing/homing.go

```diff
@@ -0,0 +1,117 @@
+package homing
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/pkg/errors"
+)
+
+const (
+	HomeUrl      = "localhost:9022"
+	stopWaitTime = 5 * time.Second
+)
+
+var ipEndpoints = []string{
+	"https://checkip.amazonaws.com/",
+	"https://ipinfo.io/ip",
+	"https://api.ipify.org/",
+}
+
+func New(svc, version string, homingLogger logger.Logger, cancel context.CancelFunc) *homingService {
```

Comment:
Instead of receiving the `version` since all the service will most likely be on the same version number, why not load if from `env` here?